app.js

```diff
@@ -1,36 +1,59 @@
+import fs from 'fs';
 import bodyParser from 'body-parser';
+import json from 'express';
 import { app, errorHandler } from 'mu';
 import { filterDeltaForInsertedType } from './lib/delta';
 import selectSubjectsWithoutUuid from './queries/select-without-uuid';
 import insertUuid from './queries/insert-uuid';
 
-const RDF_TYPE = process.env.RDF_TYPE;
-const GRAPH = process.env.GRAPH;
 
-app.post('/run', async (req, res) => {
-  const subjectsWithout = await selectSubjectsWithoutUuid(RDF_TYPE, [GRAPH]);
-  if (subjectsWithout.length > 0) {
-    console.log(`Found ${subjectsWithout.length} objects of type <${RDF_TYPE}> without a uuid. Handling now.`);
-    for (const subject of subjectsWithout) {
-      await insertUuid(subject, [GRAPH]);
+app.use(json());
+
+const CONFIG = readConfig();
+
+function readConfig() {
+  const configText = fs.readFileSync('/config/config.json', { encoding: 'utf8' });
+  const config = JSON.parse(configText);
+
+  return config;
+}
+
+app.post('/run', async (_req, res) => {
+  let uris = new Set();
+
+  for (let [type, graphs] of Object.entries(CONFIG)) {
+    const subjectsWithout = await selectSubjectsWithoutUuid(type, graphs);
+    if (subjectsWithout.length > 0) {
+      console.log(`Found ${subjectsWithout.length} objects of type <${type}> without a uuid. Handling now.`);
+      for (const subject of subjectsWithout) {
+        await insertUuid(subject, graphs);
+        uris.add(subject);
+      }
     }
   }
-  const data = subjectsWithout.map(s => {
-    return { uri: s };
-  });
 
-  return res.send({
-    data
-  });
+  let ret = [];
+
+  uris.forEach((uri) => {
+    ret.push({
+      "uri": uri,
+    })
+  })
+
+  return res.status(200).send(JSON.stringify(
+    ret
```

Comment:
If I'm not mistaken, the previous `data` key, made for the array to be wrapped within an object under the `data` key, which already comes slightly closer to the JSONAPI-compliant payload we aim for ... could you keep that here? :-)

Reply:
Ah my bad, I'll change it